File: tests/fixtures/gcs-ts/basic/no_imports.ts
// No @google-cloud/storage import — these chains should produce zero emits.

class FakeFile {
  download() { return Promise.resolve(); }
  save(_buf: Buffer) { return Promise.resolve(); }
}
class FakeBucket {
  file(_k: string) { return new FakeFile(); }
}
class FakeStorage {
  bucket(_b: string) { return new FakeBucket(); }
}

const fake = new FakeStorage();

export async function localFetch() {
  return fake.bucket('nope').file('nope').download();
}
export async function localSave() {
  return fake.bucket('nope').file('nope').save(Buffer.from(''));
}

File: tests/fixtures/lapin/basic/src/no_imports.rs
struct Fake;

impl Fake {
    pub async fn basic_publish(&self, _e: &str, _r: &str) {}
    pub async fn basic_consume(&self, _q: &str, _t: &str) {}
}

pub async fn local() {
    let f = Fake;
    f.basic_publish("nope", "nope").await;
    f.basic_consume("nope", "nope").await;
}
